src/constants/colors/primary.ts

```diff
@@ -1,66 +1,52 @@
 import PropTypes from 'prop-types';
 
-import COLORS from '.';
-
 const PRIMARY_COLORS = {
   // brand colors
-  primary: COLORS.purple100,
-  primaryTint1: COLORS.purple85,
-  primaryTint2: COLORS.purple70,
-  primaryTint3: COLORS.purple30,
-  primaryTint4: COLORS.purple15,
-  secondary: COLORS.lavender100,
-  secondaryTint1: COLORS.lavender80,
-  secondaryTint2: COLORS.lavender60,
-  secondaryTint3: COLORS.lavender10,
-  secondaryTint4: COLORS.lavender10,
-  tertiary: COLORS.purple4,
-  tertiaryTint1: COLORS.purple4,
-  tertiaryTint2: COLORS.purple4,
-  tertiaryTint3: COLORS.purple4,
-  tertiaryTint4: COLORS.purple4,
+  primary: '#332E54',
+  primaryTint1: '#524D6E',
+  primaryTint2: '#706D87',
+  primaryTint3: '#C3C0CD',
+  primaryTint4: '#E1E0E6',
+  secondary: '#A6A1E2',
+  secondaryTint1: '#B8B4E8',
+  secondaryTint2: '#CAC7EE',
+  secondaryTint3: '#EFEEF7',
+  secondaryTint4: '#EFEEF7',
+  tertiary: '#F8F8FA',
+  tertiaryTint1: '#F8F8FA',
+  tertiaryTint2: '#F8F8FA',
+  tertiaryTint3: '#F8F8FA',
+  tertiaryTint4: '#F8F8FA',
 
   // general colors
-  success: COLORS.statusGreen,
-  successBackground: COLORS.statusGreenBackground,
-  successBorder: COLORS.statusGreenBorder,
-  info: COLORS.statusPurple,
-  infoBackground: COLORS.statusPurpleBackground,
-  infoBorder: COLORS.statusPurpleBorder,
-  error: COLORS.statusRed,
-  errorBackground: COLORS.statusRedBackground,
-  errorBorder: COLORS.statusRedBorder,
-  default: COLORS.purple70,
-  defaultBackground: COLORS.purple10,
-  defaultBorder: COLORS.purple15,
-  accent: COLORS.red,
-  disabled: COLORS.purple10,
-  failure: COLORS.red,
-  hover: COLORS.purple4,
-  warning: COLORS.yellowLight,
+  default: '#706D87',
+  defaultLight: '#EDEDF0',
+  defaultBorder: '#E1E0E6',
+  success: '#2B6E33',
+  successLight: '#EAF1EB',
+  successBorder: '#DFE9E0',
+  error: '#BD200F',
+  errorLight: '#F7EAE8',
+  errorBorder: '#F3DEDC',
+  info: '#554CB3',
+  infoLight: '#EEEDF7',
+  infoBorder: '#E5E3F3',
+  black: '#000000',
+  white: '#FFFFFF',
+  transparent: 'transparent',
 
   // ui colors
```

Comment:
I don't think we're beholden to these code comments anymore and it looks like the UI Toolkit only mentions `Primary`, `Status`, and `Brand` colors. 

If you want to keep some of them that's fine, but I don't think distinguishing between `general` and `ui` is helpful anymore (if it ever was).

Reply:
yes thanks cleaned up a bit more it looks super sick now